persistence-api/pom.xml

```diff
@@ -60,7 +60,6 @@
     <dependency>
       <groupId>com.datastax.oss</groupId>
       <artifactId>java-driver-shaded-guava</artifactId>
-      <version>25.1-jre</version>
```

Comment:
no scope?

Reply:
Need to add. Curious if "provided" works here or not, will try (that is, does anything need to expose as "compile")